### ISSUE TITLE
train my own font dataset

### ISSUE THREAD

**Issue by qinb** [TAMU-VITA/ShapeMatchingGAN#7]
Hi, Thank you for you work!  
I want to  train my own handwrite dataset, the loss result as following:
```
Epoch [4/10][1757/6000]: LDadv: +0.302, LGadv: +78.938, Lrec: +0.524
Epoch [4/10][1758/6000]: LDadv: +0.314, LGadv: +78.576, Lrec: +0.518
Epoch [4/10][1759/6000]: LDadv: +0.277, LGadv: +78.604, Lrec: +0.461
Epoch [4/10][1760/6000]: LDadv: +0.260, LGadv: +77.750, Lrec: +0.441
Epoch [4/10][1761/6000]: LDadv: +0.237, LGadv: +77.608, Lrec: +0.524
Epoch [4/10][1762/6000]: LDadv: +0.204, LGadv: +78.481, Lrec: +0.516
Epoch [4/10][1763/6000]: LDadv: +0.254, LGadv: +78.299, Lrec: +0.507
Epoch [4/10][1764/6000]: LDadv: +0.308, LGadv: +79.300, Lrec: +0.587
Epoch [4/10][1765/6000]: LDadv: +0.272, LGadv: +79.734, Lrec: +0.561
Epoch [4/10][1766/6000]: LDadv: +0.255, LGadv: +79.427, Lrec: +0.497
Epoch [4/10][1767/6000]: LDadv: +0.207, LGadv: +79.826, Lrec: +0.448
Epoch [4/10][1768/6000]: LDadv: +0.208, LGadv: +79.392, Lrec: +0.501
Epoch [4/10][1769/6000]: LDadv: +0.177, LGadv: +78.863, Lrec: +0.456
Epoch [4/10][1770/6000]: LDadv: +0.225, LGadv: +78.815, Lrec: +0.495
Epoch [4/10][1771/6000]: LDadv: +0.330, LGadv: +78.598, Lrec: +0.467
Epoch [4/10][1772/6000]: LDadv: +0.341, LGadv: +79.278, Lrec: +0.497
Epoch [4/10][1773/6000]: LDadv: +0.315, LGadv: +78.820, Lrec: +0.593
Epoch [4/10][1774/6000]: LDadv: +0.207, LGadv: +78.609, Lrec: +0.428
Epoch [4/10][1775/6000]: LDadv: +0.190, LGadv: +79.434, Lrec: +0.547
Epoch [4/10][1776/6000]: LDadv: +0.314, LGadv: +79.484, Lrec: +0.582
Epoch [4/10][1777/6000]: LDadv: +0.340, LGadv: +79.534, Lrec: +0.545
Epoch [4/10][1778/6000]: LDadv: +0.240, LGadv: +79.546, Lrec: +0.484
Epoch [4/10][1779/6000]: LDadv: +0.305, LGadv: +79.679, Lrec: +0.552
Epoch [4/10][1780/6000]: LDadv: +0.389, LGadv: +78.796, Lrec: +0.527
Epoch [4/10][1781/6000]: LDadv: +0.341, LGadv: +78.778, Lrec: +0.514
Epoch [4/10][1782/6000]: LDadv: +0.353, LGadv: +78.656, Lrec: +0.610
```

the LGadv is too large,  could you tell me whether the loss is normal?     Thanks

**Comment by williamyang1991** [comment]
I have provided my training losses in ./src/ShapeMatchingGAN.ipynb
Please take it for a reference.

In my experience, the G loss of WGAN sometimes can grow very high, but I still obtain a good performance. 
You can check the quality of the generated images to see if there is a problem.

**Comment by qinb** [comment]
ok, Thanks for your reply.
